extension/content/components/pages/RecipesPage.js

```diff
@@ -21,7 +23,9 @@ import {
   useSelectedEnvironment,
   useSelectedEnvironmentAPI,
 } from "devtools/contexts/environment";
+import RecipeEditor from "devtools/components/recipes/RecipeEditor";
 import { convertToV1Recipe } from "devtools/utils/recipes";
+import { useSelectedEnvironmentAuth } from "../../contexts/environment";
```

Comment:
i'm guessing we can get rid of this import, but if not it should move up to where the other imports from this file happen and we should never use relative path imports